packages/design-tokens/src/tokens/functional/components/video-player/base.json

```diff
@@ -0,0 +1,129 @@
+{
+  "brand": {
+    "VideoPlayer": {
+      "border": {
+        "radius": {
+          "value": "var(--brand-borderRadius-medium)"
+        }
+      },
```

Comment:
Does this need to be a variable / should it be user customisable?

If it does, could we change the token to be `--brand-VideoPlayer-borderRadius` please? this will currently build as `--brand-VideoPlayer-border-radiud` (not camel-case).

Reply:
This has been updated to remove the token in the following [SHA](https://github.com/primer/brand/pull/373/commits/6f9a9c8740844ac6d50be622f1c9aab106c564dc)